IFC4x3/Properties/m/MinimumLeakageDistance_25T8FhIC540O926Lkd6LtI/Documentation.md

```diff
@@ -0,0 +1 @@
+The minimum electric current in an unwanted conductive path under normal operating conditions.
```

Comment:
this is not an adequet description of leakage/creep distance

Reply:
Need @fhulin opinion

Reply:
Changed to creepage. 08487af509e4319afa5d187d589ae5dfd92de558